### DISCUSSION TITLE
[OP Stack Deployment] Decimal Block Times

### DISCUSSION THREAD

**Discussion by fahimahmedx** [ethereum-optimism/developers#420] [Deploying the OP Stack] [answered]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

 was going through the docs for deploying an L2 rollup using the OP stack, and was looking at the [chain configurations page](https://docs.optimism.io/builders/chain-operators/management/configuration#blocks). 

![image](https://github.com/ethereum-optimism/developers/assets/68152521/94698567-d6a2-4358-a968-c902e6477063)

Does OP stack requires this number to be strictly integers? (ex. 1.5 second block times would not be a valid number).

**Comment by hash1go** [comment, marked as answer]
Hello!
op-geth has [minimal differences](https://op-geth.optimism.io/) from geth, and block times must be specified as integers.

**Comment by opfocus** [comment]
https://github.com/ethereum-optimism/optimism/blob/develop/op-chain-ops/genesis/config.go#L63
